lib/ar_transaction_changes.rb

```diff
@@ -1,7 +1,7 @@
 require "ar_transaction_changes/version"
 
 module ArTransactionChanges
-  if (ActiveRecord::VERSION::MAJOR == 4 && ActiveRecord::VERSION::MINOR >= 2) || ActiveRecord::VERSION::MAJOR > 4
+  if (ActiveRecord::VERSION::MAJOR == 4 && ActiveRecord::VERSION::MINOR >= 2)
```

Comment:
I'll have a look when was that released. If it's .0 or .1 it might be worth just dropping the line?